src/plugins/vis_type_timelion/public/timelion_vis_renderer.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { lazy } from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+
+import { KibanaContextProvider } from '../../kibana_react/public';
+import { ExpressionRenderDefinition } from '../../expressions/common/expression_renderers';
+import { VisualizationContainer } from '../../visualizations/public';
+import { TimelionVisDependencies } from './plugin';
+import { TimelionRenderValue } from './timelion_vis_fn';
+// @ts-ignore
+const TimelionVisComponent = lazy(() => import('./components/timelion_vis_component'));
+
+export const getTimelionVisRenderer: (
+  deps: TimelionVisDependencies
+) => ExpressionRenderDefinition<TimelionRenderValue> = (deps) => ({
+  name: 'timelion_vis',
+  displayName: 'Timelion visualization',
+  reuseDomNode: true,
+  render: async (domNode, { visData, visParams }, handlers) => {
```

Comment:
just curious are we really need to add async here? I don't see any await in function body. Also function doesn't return anything. This flag doesn't affect any inner async functions like e.g.  lazy TimelionVisComponent

Reply:
You're completely right, it's unnecessary! Thanks